### DISCUSSION TITLE
Issues with rendering citations properly

### DISCUSSION THREAD

**Discussion by srcorsi-USGS** [quarto-dev/quarto-cli#10768] [Q&A]
### Description

I am using Quarto in R to develop a manuscript with citations coming from a .bib file. The in-line citation and the resulting bibliography have some issues that I have not been able to resolve. I get the feeling it is due to the long Author name all in one field,  but I'm really not certain if that is correct. Any insight on this issue would be much appreciated!

````qmd
---
title: "The deicer dilema"
author: "Chet Atkins"
format: docx
echo: FALSE

bibliography: example.bib
# csl: environmental-toxicology-and-chemistry.csl  

---
  
Lester left the aircraft and smelled a strong odor of deicing agents used on the aircraft and pavement [@state_of_ohio_environmental_protection_agency_biological_1996; @u.s._environmental_protection_agency_preliminary_2000; @fisher_acute_1995]. 

# References

````

This is what I have in the example.bib file:

````
@report{u.s._environmental_protection_agency_preliminary_2000,
	location = {Washington, {DC}},
	title = {Preliminary Data Summary: Airport Deicing Operations (Revised)},
	url = {http://www.epa.gov/waterscience/guide/airport/},
	pages = {7--1--8--18},
	institution = {U.S. Environmental Protection Agency},
	author = {U.S. Environmental Protection Agency},
	date = {2000-08},
	note = {Report},
	keywords = {Central Deicing Pads, {GRV}/Sweeper Vacs, {POTW} treatment, Recycling/recovery, Storage Ponds, Storage Tanks, Anaerobic treatment, {BMP} Design criteria, {BMP} implementation guidelines, Aerobic treatment, {BMP} selection guidelines, Diversion valves / technology, Block and pump, Geosynthetics, In-line storage, Land application, O\&M costs, Online monitoring, Subsurface biodegradation, Treatment pond},
	file = {airport-deicing-pds-2000.pdf:C\:\\Users\\srcorsi\\Zotero\\storage\\EJR8HUFJ\\airport-deicing-pds-2000.pdf:application/pdf},
}

@report{state_of_ohio_environmental_protection_agency_biological_1996,
	title = {Biological and Water Quality Study of Mill Creek - 1995},
	url = {http://www.epa.ohio.gov/dsw/document_index/psdindx.aspx},
	pages = {24},
	number = {{MAS}/1996- 9-5},
	author = {State of Ohio Environmental Protection Agency},
	urldate = {2011-12-09},
	date = {1996-07-30},
}

@article{fisher_acute_1995,
	title = {The acute whole effluent toxicity of storm water from an international airport},
	volume = {14},
	url = {http://www.scopus.com/scopus/inward/record.url?eid=2-s2.0-0029001238&partner=40&rel=R5.0.4},
	pages = {1103--1111},
	number = {6},
	journaltitle = {Environmental Toxicology and Chemistry},
	author = {Fisher, D. J. and Knott, M. H. and Turley, S. D. and Turley, B. S. and Yonkos, L. T. and Ziegler, G. P.},
	date = {1995},
	note = {Journal Article},
	keywords = {Acute toxicity, {BOD}, toxicity, Airport, freshwater, environmental impact, Storm water, Deicing mixtures},
}
````

The docx file renders to the attached file example.docx

[example.docx](https://github.com/user-attachments/files/16956686/example.docx)

**Comment by mcanouil** [comment]
Could you properly format your post using code blocks for code and terminal outputs? Thanks.
If your code contains code blocks, you need to enclose it using more backticks, *i.e.*, usually four ` ```` `.
See <https://quarto.org/bug-reports.html#formatting-make-githubs-markdown-work-for-us>.

**Reply by srcorsi-USGS** [reply]
Yes. Thanks for the tip.

**Reply by mcanouil** [reply]
Could you format all your code? This includes the content of your bib file.

Additionally, you need to ensure your example is fully reproducible.
We don't have `environmental-toxicology-and-chemistry.csl`.
Note that, if you don't have issues when not using the CSL, then the issue is with this CSL and not Quarto/Pandoc.

**Reply by srcorsi-USGS** [reply]
Good point--thanks for the feedback. I didn't realize that I could render references/citations without specifying a csl. I removed that from the code, and the same problem still remains. It looks like the translation from bibtex is assuming a first and last name when there is an agency name as Author. Perhaps there is a better way for bibtex to represent this for a different interpretation by Quarto? Here is the rendered text without a csl specified. 

[example.docx](https://github.com/user-attachments/files/16964681/example.docx)

**Reply by mcanouil** [reply]
The issue is not Quarto or the CSL, the issue is that your bib latex code is not valid.
I just noticed the missing brackets.

This is not a valid author.
```
author = {State of Ohio Environmental Protection Agency}
```

This is valid:
```
author = {{State of Ohio Environmental Protection Agency}}
```

**Reply by srcorsi-USGS** [reply]
OK--thanks for the info. Much appreciated! I ran it on my end, and it works fine with these extra brackets. 

I had exported those references from Zotero, so will to mention it to those folks and do some manual manipulations for now.

Best regards

**Reply by mcanouil** [reply]
You should double check how you export from Zotero (biblatex or bibtex) and/or how you imported the references.
There are other places where "internal brackets" are needed in biblatex/bibtex references.

**Reply by srcorsi-USGS** [reply]
Good suggestion. I have tried several formats coming out of Zotero, all with the same result. The answer from Zotero forum suggests that the CSL json format would be more seamless. I have tried that now with ".json", and it works well.
